Dockerfile

```diff
@@ -27,10 +27,6 @@ WORKDIR /go/src/velero-plugin-for-microsoft-azure
 RUN export GOARM=$( echo "${GOARM}" | cut -c2-) && \
     CGO_ENABLED=0 go build -v -o /go/bin/velero-plugin-for-microsoft-azure ./velero-plugin-for-microsoft-azure
 
-FROM busybox:1.34.1 AS busybox
-
-FROM gcr.io/distroless/base-debian10:nonroot
+FROM scratch
```

Comment:
@ywk253100 
Not sure about the reason we use distroless image in plugin.
Is it OK changing to scratch?

Reply:
@ywk253100 the vulnerability was in libc6_2.28-10+deb10u1_amd64.deb but it was some time ago (13-04-2022).
Should I keep scratch but drop the go cp? Maybe busybox could be used directly instead of copying cp (which already contains much more then cp)?
Do you think the Velero project could be interested to provide the cp command for all the plugins?

Reply:
@yvespp Please keep the scratch but drop the go cp. We don't want to maintain a go cp command if it isn't necessary at this moment.

I don't undertand what doest `Maybe busybox could be used directly instead of copying cp (which already contains much more then cp)?` mean. How could the busybox be used directly? The `busybox` and `cp` are the same binary with different names, and you cannot use `cp` as other commands. Correct me if I'm wrong

Reply:
@ywk253100 I mean instead of using scratch use busybox as the image for the plugin.
cp is actually a binary with all system tools inside. Depending by which name it's called (arg $0) the corresponding function is called. You can test this like this:
```
docker run -it busybox
cd $HOME
cp /bin/cp cp
mv ./cp wget
./wget google.com
mv ./wget sh
./sh -c "echo hello"
```
So basically by copying cp the whole busybox is copied.
In the wiki page I linked it's described how it works and what commands it contains: https://en.wikipedia.org/wiki/BusyBox#Single_binary

Reply:
@yvespp Understood. 
I suggest let's use `scratch` and copy the `cp` from `busybox` to make sure the image contains only the necessary packages. The `busybox` may or may not contain other files/packages aren't needed by plugins which may introduce new CVEs. 

And copying from `busybox` is easy for us to replace, maybe we'll use our own `cp` command in future releases just like you do in the PR. But for now let's copy from `busybox`